### PR TITLE
Backend for reminder banner

### PR DESCRIPTION
**Pull request by ashlilili**
1. Add profile completion percentage and last update in mock server user schema, add an api getUserProfileInfo  for getting these two fields
2. The app queries mock backend for user info and renders the banner accordingly
3. (small fix) FirebaseCrashlytics doesn't support web app, so I added an extra condition 

next step would be to update onError to use crashHandler.

### CODE REVIEW

File: mm-mock-server/src/mocks/util/generators.ts [hunk 1]
```diff
@@ -92,6 +92,20 @@ export function generateChannelInboxItemMessage(channelId: string, sender: any)
...
+    // Returns a random integer between 0 and 100
+    return  Math.floor(Math.random() * 100); 
+}
+
+
+export function generateUserLastUpdateTime() {
+    // Returns a random date time from one year before to now
+    var startDate = (new Date(new Date().setFullYear(new Date().getFullYear() - 1))).valueOf(); 
+    var endDate = (new Date()).valueOf();
+    return new Date(startDate + Math.random() * (endDate - startDate));
```
Comment (m-rosario) [inline mm-mock-server/src/mocks/util/generators.ts:105]: Nit: I believe the faker library can help with generating this random data with less code.

**Review by m-rosario — APPROVED** [review]
Looks great, thanks!

File: mm-mock-server/src/mocks/util/generators.ts (hunk 1 above)
Comment (ashlilili) [reply mm-mock-server/src/mocks/util/generators.ts]: Great tip! Changed to faker library.

**Review by m-rosario — APPROVED** [review]

**Review by andersan — COMMENTED** [review]
Still had this tab opened and realized i never sent these review comments from yesterday. Already implemented a fix in #38 and implemented a mock for profile completion percentage on the User resolver in the mmdata backend, too.

File: lib/data/models/user/user_api.dart [hunk 2]
```diff
@@ -59,3 +59,12 @@ const String kGetAllUsers = """
     }
   }
 """;
+
+const String kGetUserProfileInfo = """
+  query Q {
+    getUserProfileInfo(){
```
[the remark this answers is not in the document]
Comment (ashlilili) [reply lib/data/models/user/user_api.dart:65]: I see , in this PR I added the two fields under User. I agree that we should reuse updatedAt if it's doing the same thing as lastUpdateTime, and add the field in the User resolver! Thanks for catching this

**Pull request merged**